### PR TITLE
bug: fix install from pypi source release

### PR DESCRIPTION
**Pull request by VoVAllen**
Signed-off-by: Jinjing.Zhou <allenzhou@tensorchord.ai>

Related: #1061

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -28,12 +29,15 @@ def build_envd_if_not_found():
         logging.info("envd not found. Build from scratch")
         try:
```
Comment (kemingy) [inline setup.py:30]: I would prefer to use https://docs.python.org/3/library/os.path.html#os.path.isfile

File: setup.py [hunk 2]
```diff
@@ -53,6 +57,17 @@ def build_extension(self, ext: Extension) -> None:
...
 
+class SdistCommand(distutils.command.sdist.sdist):
+    def run(self):
+        errno = subprocess.call(["make", "generate-git-tag-info"])
+        assert errno == 0, "Failed to generate git tag info"
+        if not os.path.exists(".GIT_TAG_INFO"):
+            logging.warning(".GIT_TAG_INFO not found")
+        else:
+            logging.info(".GIT_TAG_INFO generated")
+        distutils.command.sdist.sdist.run(self)
```
Comment (kemingy) [inline setup.py:68]:
```suggestion
        super().run()
```

File: setup.py [hunk 3]
```diff
@@ -53,6 +57,17 @@ def build_extension(self, ext: Extension) -> None:
...
 
+class SdistCommand(distutils.command.sdist.sdist):
+    def run(self):
+        errno = subprocess.call(["make", "generate-git-tag-info"])
+        assert errno == 0, "Failed to generate git tag info"
+        if not os.path.isfile(".GIT_TAG_INFO"):
+            logging.warning(".GIT_TAG_INFO not found")
+        else:
+            logging.info(".GIT_TAG_INFO generated")
+        super.run(self)
```
[the remark this answers is not in the document]
Comment (kemingy) [reply setup.py:68]: BTW, it should be `super()` not `super`

File: setup.py [hunk 4]
```diff
@@ -53,6 +57,17 @@ def build_extension(self, ext: Extension) -> None:
...
 
+class SdistCommand(distutils.command.sdist.sdist):
+    def run(self):
+        errno = subprocess.call(["make", "generate-git-tag-info"])
+        assert errno == 0, "Failed to generate git tag info"
+        if not os.path.isfile(".GIT_TAG_INFO"):
+            logging.warning(".GIT_TAG_INFO not found")
+        else:
+            logging.info(".GIT_TAG_INFO generated")
+        super.run()
```
Comment (kemingy) [inline setup.py:68]:
```suggestion
        super().run()
```

Comment (VoVAllen) [reply setup.py]: :<

**Review by kemingy — APPROVED** [review]
/lgtm

**Pull request merged**